### DISCUSSION TITLE
Problem with 512 gb disk

### DISCUSSION THREAD

**Discussion by agt361** [mcguirepr89/BirdNET-Pi#531] [General]
I have had BirdNET-Pi ruinning on a 256 GB disk and a 128 GB disk.  I thought I would improve the data base life with a bigger disk.
Now I get 'no module named 'tzlocal' in the log file and now bird species loaded.  Any ideas? Thanks.

**Comment by mcguirepr89** [comment]
My only idea is that whatever happened between these two sentences is the culprit:
> I thought I would improve the data base life with a bigger disk.

>Now I get 'no module named 'tzlocal' in the log file and now bird species loaded. Any ideas?

If you could please provide a lot more information, e.g.:
- Are you saying that you swapped out your hard disk and now the system is broken?
- Are you saying that you tried a new installation and it failed?
- Are you saying that you have an external HDD mounted that is failing?
- Are you saying that you made changes to your system or that you are starting a new one?
- What sorts of steps did you perform for _any_ portion of your setup that deviates from the installation guide?

Any information like that would be both needed and helpful.

My best,
Patrick

PS
There is no relationship between the database size and the disk size in BirdNET-Pi -- the database grows indefinitely on all systems.

**Reply by agt361** [reply]
I started with a brand new sd card (512).  After the usual Caddy hiccup - solved by retrying - I got to the normal BirdNET screen.  The log showed perpetual tzlocal errors.  Rebooted twice with no luck and no bird recognition.

So I started from scratch again - this time (after a Caddy message as usual) the system worked but very very slowly.  Rebooted and - wow - it works perfectly. 

Can think of now difference between my installations.  I now think that it is a good idea to do a reboot of the Pi before any installation.

I am so sorry to have troubled you.

**Reply by agt361** [reply]
"no difference" and both installations exactly as described in your instaructions.
